store/src/main/java/in/testpress/store/ui/ProductDetailFragment.kt

```
@@ -115,14 +159,50 @@ class ProductDetailFragment : Fragment(), EmptyViewListener {
+                discountPrompt.apply {
+                    setOnClickListener {
+                        if (discountContainer.visibility == View.GONE) {
+                            discountContainer.visibility = View.VISIBLE;
+                            discountPrompt.visibility = View.GONE;
+                        }
+                    }
+                }
+
+                couponInputEditText.apply {
+                    addTextChangedListener(object : TextWatcher {
+                        override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)